tests/test_scripts.py

```diff
@@ -687,21 +689,29 @@ def test_analyze_gather_tb(tmpdir: str):
     assert run.result["n_tb_dirs"] == 2
 
 
-def test_convert_trajs_in_place(tmpdir: str):
+def test_convert_trajs(tmpdir: str):
+    """Tests that convert_trajs is idempotent and does not change the data."""
     shutil.copy(CARTPOLE_TEST_ROLLOUT_PATH, tmpdir)
     tmp_path = os.path.join(tmpdir, os.path.basename(CARTPOLE_TEST_ROLLOUT_PATH))
     exit_code = subprocess.call(
```

Comment:
Could we run this directly via sacred instead of subprocess? This way we could also check if the deprecation warning is raised, ensuring that the test really converts from the old format to the new format.

Reply:
I've added a check that original input file is pickle format and output is in npz format.

Unfortunately `convert_trajs` is not written in Sacred. We could call the `main()` method directly and mock `sys.argv`, or call the `update_traj_file_in_place` method directly. Perhaps ideally we'd do both unit and integration test, but feels a bit overkill to me for such a simple piece of code.

Reply:
I would prefer mocking `sys.argv` or even calling `update_traj_file_in_place` directly. I think it would make the test easier to read and mostly also easier to debug. I try to avoid subprocesses in tests at all costs since it makes debugging so much more painful.